python/ray/util/spark/databricks_hook.py

```diff
@@ -44,13 +44,14 @@ def display_databricks_driver_proxy_url(spark_context, port, title):
     orgId = commandContextTags.apply("orgId")
     clusterId = commandContextTags.apply("clusterId")
 
-    template = "/driver-proxy/o/{orgId}/{clusterId}/{port}/"
-    proxy_url = template.format(orgId=orgId, clusterId=clusterId, port=port)
+    proxy_link = f"/driver-proxy/o/{orgId}/{clusterId}/{port}/"
+    proxy_url = f"https://dbc-dp-{orgId}.cloud.databricks.com{proxy_link}"
 
+    _logger.info(f"View the dashboard at {proxy_url}")
```

Comment:
shall we use "print" so that it will always show the message even higher log level is set ?

Reply:
Happy to make this change - I think it makes sense to remain consistent with the other logging patterns used, which seem to be either a module-level logger as we have in `databricks_hook`, or the custom CLI logger, which looks like it uses `print` explicitly.

What are your thoughts on importing and reusing the CLI logger used in `scripts.py`? Something like

```py
from ray.autoscaler._private.cli_logger import cf, cli_logger
proxy_url = get_databricks_proxy_url()

cli_logger.print("To monitor and debug Ray from Databricks, view the dashboard at ")
cli_logger.print(
    "  {}".format(
        cf.bold(proxy_url),
    )
)
```

Reply:
I think for now we can just use `print` which is what cli_logger uses under the hood.

Reply:
Great, I've swapped to using print :). Just for my understanding, is there a reason you prefer not use one of the existing loggers (either from `logging` or the `cli_logger` so we can take advantage of the bolded formatting)?

A little different from the original PR, I've kept the print over 2 lines, [consistent with the standard dashboard URL print](https://github.com/ray-project/ray/blob/master/python/ray/scripts/scripts.py#L831), and specifically mention Databricks. My reasoning is to provide some explanation to the user why there are two separate messages regarding the dashboard. Happy to change/simplify this too if you think it is too much.